internal/commands/sbomtest/sbomloader.go

```diff
@@ -0,0 +1,52 @@
+package sbomtest
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"os"
+)
+
+func IsSBOMJSON(b []byte) bool {
+	var sbom interface{}
+	err := json.Unmarshal(b, &sbom)
+	if err != nil {
+		return false
+	}
+
+	if _, ok := sbom.(map[string]interface{}); !ok {
+		return false
+	}
+
+	return true
+}
```

Comment:
if we declare `sbom` as `map[string]interface{}` from the start, would that make the type assertion redundant?

```suggestion
func IsJSON(b []byte) bool {
	var sbom map[string]interface{}
	if err := json.Unmarshal(b, &sbom); err != nil {
		return false
	}

	return true
}
```

Reply:
Yes, thank you. It can be reduced too

```
func IsSBOMJSON(b []byte) bool {
	var sbom map[string]interface{}
	err := json.Unmarshal(b, &sbom)
	return err == nil
}
```